File: 1.Basic-Concepts/a.arrays/arrays.js
const myArray = [1, 20, 300, 4000, 50000];

console.log('============== For loop ================');
for (let i = 0; i < myArray.length; i++) {
  console.log(`Index - ${i}, Item : ${myArray[i]}`);
}

console.log('============== For each ================');
myArray.forEach((num, idx) => {
  console.log(`Index - ${idx} ,Item : ${num} `);
});

File: 1.Basic-Concepts/a.arrays/arrays.cpp
#include <iostream>

int main()
{
    int my_array[] {2,5,7,8,7};

    std::cout << " Size of array : " << sizeof(my_array) << "\n";
    for(int i = 0; i < 5; i++)
    {
        std::cout << " Arr val : " << my_array[i] << "\n";
    }

    std::cout << "========= For each loop ====="<<std::endl;
    for(int value:my_array)
    {
        std::cout << "value : " << value << std::endl;
    }

    return 0;
}